src/lib/SCEditor.js

```diff
@@ -355,7 +355,7 @@ define(function (require) {
 				}
 
 				if (options.autoExpand) {
-					base.expandToContent();
+					base.expandToContent(options.ignoreMaxHeight);
```

Comment:
Thanks, I missed that one.  I'll add it and test it. Then add a comment for review after I push again.